client/src/modules/cohort/model/selectors.js

```diff
@@ -54,5 +54,5 @@ export const getArchivedCohorts = createSelector(
 
 export const getMembers = createSelector(
   getCohortDetails,
-  cohort => cohort && cohort.members
+  cohort => (cohort && cohort.members) || {}
```

Comment:
This can cause unnecessary re-renderings. You can extract `const emptyObject = {}` and then use it as a fallback to avoid constant re-renderings.